examples/Cpp/GoogleTest/lib/googletest/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(FetchContent)
+
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest.git
+  GIT_TAG        f8d7d77c06936315286eb55f8de22cd23c188571 # release-1.14.0
```

Comment:
Does CMake support symbolic refs like tags instead of heavy hashes?

Reply:
What do mean by heavy hash?